addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -21,7 +21,7 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
-    toggleLinkTools(options) {
+toggleLinkTools(options = {}) {
```

Comment:
I don't think you meant to outdent this line.

Also two little comments about the commit message: "led" instead of "lead", and the syntax for the link to commit is `[1]: your_commit` (you're missing the `:`).

Other than that lgtm.

Reply:
Oh and in the meantime there's also [opw-3151829](https://www.odoo.com/web#id=3151829&cids=1&model=project.task&view_type=form) to mention